### PR TITLE
Add Metacritic

### LINKED ISSUE #10065: Request: Metacritic
### Brand Name

Metacritic

### Website

https://www.metacritic.com/

### Popularity Metric

The Similarweb rank is 4,488. See https://www.similarweb.com/website/metacritic.com

### Official Resources for Icon and Color

svg: from website header

```html
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><path d="M36.978 19.49a17.49 17.49 0 1 1 0-.021" fill="#000"></path> <path d="m17.209 32.937 3.41-3.41-6.567-6.567c-.276-.276-.576-.622-.737-1.014-.369-.783-.53-2.004.369-2.903 1.106-1.106 2.58-.645 4.009.784l6.313 6.313 3.41-3.41-6.59-6.59c-.276-.276-.599-.691-.76-1.037-.438-.898-.415-2.027.392-2.834 1.129-1.129 2.603-.714 4.24.922l6.128 6.129 3.41-3.41L27.6 9.274c-3.364-3.364-6.52-3.249-8.686-1.083-.83.83-1.337 1.705-1.59 2.696a6.71 6.71 0 0 0-.092 2.81l-.046.047c-1.66-.691-3.549-.277-5 1.175-1.936 1.935-1.866 3.986-1.636 5.184l-.07.07-1.681-1.36-2.95 2.949c1.037.945 2.282 2.097 3.687 3.502l7.673 7.673Z" fill="#F2F2F2"></path> <path d="M19.982 0A20 20 0 1 0 40 20v-.024A20 20 0 0 0 19.982 0Zm-.091 4.274A15.665 15.665 0 0 1 35.57 19.921v.018A15.665 15.665 0 1 1 19.89 4.274Z" fill="#FFBD3F"></path></svg>
```

color: #FFBD3F from website header

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by SVendittelli**
metacritic

**Issue:** closes #10065

**Similarweb rank:** [4,488](https://www.similarweb.com/website/metacritic.com)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Logo and color from the [website header](https://www.metacritic.com).

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -8585,6 +8585,11 @@
             "hex": "509EE3",
             "source": "https://www.metabase.com"
         },
+        {
+            "title": "Metacritic",
+            "hex": "FFBD3F",
+            "source": "https://www.metacritic.com"
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:8591]:
```suggestion
            "source": "https://www.metacritic.com",
            "guidelines": "https://developer.iva-api.com/docs/v2/metacritic-brand-guidelines"
```

**Review by PeterShaggyNoble — APPROVED** [review]
SVG looks perfect to me, @SVendittelli, nice work 👍🏻 

For the colour, though: the black seems to be the dominant colour on their site, rather than the yellow. What do you think?

The only other suggestion I would have is the inclusion of their brand guidelines below.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -8585,6 +8585,11 @@
             "hex": "509EE3",
             "source": "https://www.metabase.com"
         },
+        {
+            "title": "Metacritic",
+            "hex": "FFBD3F",
```
Comment (SVendittelli) [inline _data/simple-icons.json:8590]:
```suggestion
            "hex": "000000",
```

**Comment by SVendittelli** [comment]
@PeterShaggyNoble I'm happy to change the colour and add that link 👍

**Pull request merged**